tools/semgrep/ci/fmt-printf.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: fmt.Printf
+    languages: [go]
+    message: fmt.Printf is forbidden outside of cmd and test files
+    pattern: fmt.Printf
+    severity: ERROR
+    paths:
+      exclude:
+        - "*test*.go"
+        - "cmd/*.go"
+        - "cmd/**/*.go"
```

Comment:
Should this be `command` instead of `cmd`? I don't see a `cmd` directory.

Reply:
Oh, I see a bunch under `builtin`, but I think these paths apply from the root, no?

Reply:
These don't apply from the root.

For example, if I remove them:

```
$ make semgrep-ci
semgrep --error --include '*.go' --exclude 'vendor' -f tools/semgrep/ci .
...
  serviceregistration/kubernetes/client/cmd/kubeclient/main.go
     tools.semgrep.ci.fmt.Printf
        fmt.Printf is forbidden outside of cmd and test files

         68┆ fmt.Printf("pod: %s\n", b)
```

Reply:
Ah, TIL. So `cmd` is on purpose then. 👍